### ISSUE TITLE
If I use another dataset with different dimensions

### ISSUE THREAD

**Issue by amirhfarzaneh** [naturomics/CapsLayer#9]
Do I have to only change the dimension in line 124 of main.py? Is that the only dimension needs to be changed?

I'm trying to train this with the FER2013 face emotion dataset which has images of size 48x48. I load the dataset properly and I have changed the line 124 accordingly but I don't see a good trend in training like raising accuracy!

**Comment by Allong12** [comment]
You will need to change the parameters on line 124 of what image size to expect, but then you will need to ensure your custom data is being loaded in; have you properly set up your custom dataset loading in `capslayer/utils.py`?

main.py will [load the dataset](https://github.com/naturomics/CapsLayer/blob/master/models/main.py#L46) defined by cfg.dataset and call the [appropriate dataset handler](https://github.com/naturomics/CapsLayer/blob/master/capslayer/utils.py#L104) in utils.py, for example [`load_mnist`](https://github.com/naturomics/CapsLayer/blob/master/capslayer/utils.py#L30)

**Comment by Allong12** [comment]
Sorry, I didn't properly read that you already have set up data loading

I believe I had to make some tweaks in my fork in order to load RGB images, and also to change the number of logits on the output, specifically the `num_outputs` argument in the [output FC layer in vectorCapsNet.py](https://github.com/naturomics/CapsLayer/compare/master...Allong12:master#diff-cdd880afc44ad802a114ff1453a432d1R76).

The current master of my fork is currently working for the binary classification of gender on the RGB, 299x299px [LFW Faces Dataset](http://vis-www.cs.umass.edu/lfw/) up to about 95% accuracy, but decoded reconstruction isn't great, I might need to investigate some larger Fully Connected layers in the decoding stage
